actors/datacap/src/lib.rs

```diff
@@ -282,7 +288,13 @@ impl Actor {
             .transaction(|st: &mut State, rt| {
                 let allowed = to_address == st.governor;
```

Comment:
I get that we don't need it based on current callers.  But for consistency it seems like we should use the same boolean expression as above for `allowed`.

Reply:
I considered it, but if some actor had allowance to transact _from_ the verified registry, that would probably be a bug. So I left this as defence-in-depth.